client/src/components/Coin.tsx

```diff
@@ -0,0 +1,14 @@
+import React from "react";
+
+const Coin = ({ url, name, price }: { url: string, name: string, price: number }) => {
```

Comment:
Could you format the props similar to how it is in SignUp/SignIn? It makes the types less verbose and easier to read in the component function signature